static/gui/static/js/components/Navbar.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import { withCookies, Cookies } from 'react-cookie';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as Icons from '@fortawesome/free-solid-svg-icons';
+import LoginModal from './Login/LoginModal.js';
+import RegisterModal from './Login/RegisterModal.js';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+
+/**
+ * New component for the Navbar on top
+ * This switches pages and renders login info
+ */
+const Navbar = (props) => {
+  const [loginEmail, setLoginEmail] = useState(props.cookies.get('current_user_email') || "");
+  const [isLoggedIn, setIsLoggedIn] = useState(loginEmail !== "");
+  const [isSideNavOpen, setIsSideNavOpen] = useState(false);
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  useEffect(() => {
+    const current_user_email = props.cookies.get('current_user_email') || "";
+
+    setIsLoggedIn(current_user_email !== "");
+    setLoginEmail(current_user_email);
+
+  }, [document.cookie]);
+
+
+  function handleLogout(e) {
+    console.log("logout")
+    if (activeIndex === 2) {
+      document.querySelector("#analytics-link").classList.remove("active");
```

Comment:
I just realized that lines 36 and 37 are not required. Also, since we are using react router now I can add a couple lines of code so that it actually redirects on logout if in analytics or history.